package.json

```diff
@@ -74,6 +74,8 @@
     "yamljs": "^0.3.0"
   },
   "dependencies": {
+    "ivy-codemirror": "^2.1.0",
```

Comment:
Can this be loaded in just your engine instead of the whole app

Reply:
Can't setup the related settings for codemirror in `ember-cli-build.js` under engine.